src/logger.ts

```diff
@@ -0,0 +1,72 @@
+import { Client } from "./client"
+
+export type LoggerLevel = "trace" | "debug" | "info" | "warn" | "error"
```

Comment:
What about implementing this one as an enum?

Reply:
My initial implementation for this was an enum! I'm willing to be convinced that it should be an enum.

I decided against it because I thought it was nicer for users (especially JavaScript users) to not have to import the LoggerLevel enum:

```js
// with enum
import { Appsignal, LoggerLevel } from "@appsignal/nodejs";

Appsignal.logger("groupname", LoggerLevel.trace)

// without enum
import { Appsignal } from "@appsignal/nodejs";

Appsignal.logger("groupname", "trace")
```

Reply:
True, I tend to forget that not everyone is using TS.